### PR TITLE
Add GH action to do a basic build of the package

### PR DESCRIPTION
**Pull request by thomaslee**
This is more of a smoke test `dpkg-buildpackage` than a full-fledged `sbuild && lintian && autopkgtest` but it's something. Maybe we can shore it up more later.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+on:
+  push:
+    branches:
+      - main
```
Comment (tmancill) [inline .github/workflows/test.yml]: What does "main" refer to here?  Is it an alias for the Default branch configured for a repo?  (I'm just curious.)

**Review by tmancill — APPROVED** [review]
Looks good to me.  I'm interested to see how this works.

File: .github/workflows/test.yml (hunk 1 above)
Comment (thomaslee) [reply .github/workflows/test.yml]: Ugh yes this is me not paying attention while trying to adapt examples, thanks for pointing this out.

**Comment by thomaslee** [comment]
>  I'm interested to see how this works.

You can see a successful run for this PR by clicking "Details" here if you like. :) It was pretty easy modulo my own copy/paste/edit ineptitude.

image

File: .github/workflows/test.yml (hunk 1 above)
Comment (thomaslee) [reply .github/workflows/test.yml]: Changed to `debian/master`

**Comment by thomaslee** [comment]
See also https://github.com/thomaslee/capnproto-debian/actions

**Pull request merged**